src/main/java/cubicchunks/asm/mixin/core/client/MixinRenderGlobal.java

```diff
@@ -23,193 +23,213 @@
-        }
-
-        Cube cube = ((IColumn) chunk).getCube(Coords.blockToCube(position.getY()));
-        if (cube instanceof BlankCube) {
-            return ClassInheritanceMultiMapFactory.EMPTY_ARR;
-        }
-
-        return new ClassInheritanceMultiMap[]{cube.getEntityContainer().getEntitySet()};
+    @Redirect(method = "renderEntities", at = @At(value = "INVOKE", target = "Lnet/minecraft/world/chunk/Chunk;getEntityLists()[Lnet/minecraft/util/ClassInheritanceMultiMap;"))
+    public ClassInheritanceMultiMap<Entity>[] onGettingClassInheritanceMultiMap(Chunk chunk) {
+        return dummyEntityStorageArray;
     }
 
     /**
      * Overwrite getRenderChunk(For)Offset to support extended height.
      *
      * @author Barteks2x
-     * @reason Remove hardcoded height checks, it's a simple method and doing it differently would be problematic and
-     * confusing (Inject with local capture into BlockPos.getX() and redirect of BlockPos.getY())
+     * @reason Remove hardcoded height checks, it's a simple method and doing it differently would be problematic and confusing (Inject with local capture into BlockPos.getX() and redirect of BlockPos.getY())
      */
     @Nullable
     @Overwrite
     private RenderChunk getRenderChunkOffset(BlockPos playerPos, RenderChunk renderChunkBase, EnumFacing facing) {
         BlockPos blockpos = renderChunkBase.getBlockPosOffset16(facing);
-        return MathHelper.abs(playerPos.getX() - blockpos.getX()) > this.renderDistanceChunks * 16 ? null :
-                MathHelper.abs(playerPos.getY() - blockpos.getY()) > this.renderDistanceChunks * 16 ? null :
-                        MathHelper.abs(playerPos.getZ() - blockpos.getZ()) > this.renderDistanceChunks * 16 ? null :
-                                this.viewFrustum.getRenderChunk(blockpos);
+        int size = RenderVariables.getRenderChunkSize();
+        return MathHelper.abs(playerPos.getX() - blockpos.getX()) > this.renderDistanceChunks * size ? null
+                : MathHelper.abs(playerPos.getY() - blockpos.getY()) > this.renderDistanceChunks * size ? null
+                        : MathHelper.abs(playerPos.getZ() - blockpos.getZ()) > this.renderDistanceChunks * size ? null
+                                : this.viewFrustum.getRenderChunk(blockpos);
     }
 
-    @ModifyConstant(
-            method = "renderWorldBorder",
-            constant = {
-                    @Constant(doubleValue = 0.0D),
-                    @Constant(doubleValue = 256.0D)
-            },
-            slice = @Slice(from = @At(value = "HEAD"), to = @At(value = "INVOKE", target = "Lnet/minecraft/client/renderer/Tessellator;draw()V")), require = 2)
+    @ModifyConstant(method = "renderWorldBorder", constant = {
+            @Constant(doubleValue = 0.0D),
+            @Constant(doubleValue = 256.0D)
+    }, slice = @Slice(from = @At(value = "HEAD"), to = @At(value = "INVOKE", target = "Lnet/minecraft/client/renderer/Tessellator;draw()V")), require = 2)
```

Comment:
Why reformat that?

Reply:
Usually I'm using `ctrl`+`a` and `ctrl`+`f` mindlessly.

Reply:
`ctrl`+`a` + `ctrl`+`f`

Reply:
Configure your IDE to reformat only code that has been marked as changed in git. If you use eclipse, it may not be possible, but there is no way to make both IDEs format code exactly the same.